Sources/PackageSets/Model/CVE.swift

```diff
@@ -0,0 +1,22 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2020 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import struct Foundation.URL
+
+extension PackageSetsModel {
+    /// A representation of Common Vulnerabilities and Exposures (CVE)
+    public struct CVE {
+        /// CVE identifier
+        public let identifier: String
```

Comment:
Is this free-form, or documented to be of a certain form?  If the latter, it would be good into include a reference to the description of how it needs to be formatted.

Reply:
it is free form